playbook/app/pb_kits/playbook/pb_filter/docs/_filter_default.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { Button, Filter, Flex, Select, TextInput } from '../../'
 
-const FilterDefault = (props) => {
+const FilterDefault = () => {
```

Comment:
Looks like spread props were removed. This will disable dark mode for this example

Reply:
<img width="1469" alt="Screen Shot 2021-01-11 at 3 13 57 PM" src="https://user-images.githubusercontent.com/28073714/104233302-a7bf8d00-541f-11eb-8f6d-84f9ae4e6564.png">

This is what you were looking for here, right?